src/sentry/static/sentry/app/routes.jsx

```diff
@@ -255,29 +255,29 @@ function routes() {
         name="Incident Rules"
         path="incident-rules/"
         componentPromise={() =>
-          import(/* webpackChunkName: "ProjectIncidentRules" */ 'app/views/settings/projectIncidentRules')
+          import(/* webpackChunkName: "OrganizationIncidentRules" */ 'app/views/settings/organizationIncidentRules')
         }
         component={errorHandler(LazyLoad)}
       >
         <IndexRoute
           componentPromise={() =>
-            import(/* webpackChunkName: "IncidentRulesList" */ 'app/views/settings/projectIncidentRules/list')
+            import(/* webpackChunkName: "IncidentRulesList" */ 'app/views/settings/organizationIncidentRules/list')
```

Comment:
Why not `incidentRules` if there aren't going to be project incident rules the prefix doesn't matter that much.